src/lightning_app/core/work.py

```diff
@@ -540,10 +540,6 @@ def _aggregate_status_timeout(self, statuses: List[Dict]) -> WorkStatus:
         status = {**timeout_statuses[-1], "timestamp": statuses[0]["timestamp"]}
         return WorkStatus(**status, count=len(timeout_statuses))
 
-    def load_state_dict(self, state):
```

Comment:
For confirmation: are we removing reloading from Work and expect this to only happen in the parent Flow?

What I'm concerned about is not really the loading, but the serializing. The work will need a way to produce its closest resumable state, which I'm expecting to be specific to the logic of the work itself in the general case.

Reply:
The plan is to have a `persist` and `restore` methods for the works as the state isn't fully part of the captured and a start_restart="latest|run_start". On reload, the runtime will select the right state and reload them.

Reply:
Ok, agreed